docs/2.8.0/docs/kubernetes/06-advanced-topics/high-availability.rst

```diff
@@ -1,108 +1,90 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-How Do We Achieve High Availability for Daml Enterprise on Kubernetes?
-######################################################################
+How to achieve high availability for Daml Enterprise on Kubernetes
```

Comment:
This is not really a 'how to', but rather 'here is how HA is achieved'. Hence, I'd suggest changing this to:
```suggestion
How is high availability achieved for Daml Enterprise on Kubernetes
```